ophyd/signal.py

```diff
@@ -33,9 +32,12 @@ class Signal(OphydObject):
         The absolute tolerance associated with the value
     rtolerance : any, optional
         The relative tolerance associated with the value, used in
-        set_and_wait as follows:
-            absolute(setpoint - readback) <= (tolerance + rtolerance *
-                                              absolute(readback))
+        set_and_wait as follows
+
+        .. math::
+
+          |setpoint - readback| \leq (tolerance + rtolerance * |readback|)
```

Comment:
but it looks so pretty in the rendered docs.  I could be convinced to go back to <= or use unicode here.